### ISSUE TITLE
3060Ti Dag has been damaged

### ISSUE THREAD

**Issue by cwkoomy** [develsoftware/GMinerRelease#788]
Dag has been damaged, check overclocking settings
gminer or anyone can help me ? my 3060ti hynix memory getting dag has been damaged, check overclocking settings, 
hiveos already lowest my oc until 0core 0mem, still get error, is it my card vram faulty ?😢. before 80% unlock, my oc up to 2700mem is never get this error. my card is gigabyte gaming oc pro rev3.
nbminer,t-rex,lol, after 100% unlock ,this card cannot be mining, few minute will get error

**Comment by yifan4bao** [comment]
Hello, my 3060ti has the same problem. Have you solved it?

**Comment by cwkoomy** [comment]
> Hello, my 3060ti has the same problem. Have you solved it?

no, still waiting for help, i dunno how to solve it . try many way, disable GUI, clear all oc before run mining, all have same error. (hiveos)

**Comment by yifan4bao** [comment]
Yes, I have also made various attempts, and the results are the same. There is no problem in testing the graphics card, so experts can come up with solutions

**Comment by cwkoomy** [comment]
> Yes, I have also made various attempts, and the results are the same. There is no problem in testing the graphics card, so experts can come up with solutions

i think this miner make our card faulty, my card plug in windows test memory also fault. playing game , if memory up to 7000mhz few minute will crash... so sad

**Comment by yifan4bao** [comment]
Yes, I also suspect that miner caused it. There is no problem testing the software video memory with mats. Which software do you use to test the video memory? I will also test it.

**Comment by yifan4bao** [comment]
Hello, friend, has your problem been solved? I think there is something wrong with the video memory particles. My two cards are Hynix.

**Comment by zulu1905** [comment]
@cwkoomy can you share gpuz secreenshot

**Comment by cwkoomy** [comment]
> Hello, friend, has your problem been solved? I think there is something wrong with the video memory particles. My two cards are Hynix.

no, i will bring back to warranty check.

**Comment by huyao8023** [comment]
same problem. 😢

**Comment by cwkoomy** [comment]
> Dag has been damaged, check overclocking settings gminer or anyone can help me ? my 3060ti hynix memory getting dag has been damaged, check overclocking settings, hiveos already lowest my oc until 0core 0mem, still get error, is it my card vram faulty ?😢. before 80% unlock, my oc up to 2700mem is never get this error. my card is gigabyte gaming oc pro rev3. nbminer,t-rex,lol, after 100% unlock ,this card cannot be mining, few minute will get error

My card already warranty back. the problem is vram faulty. so next time this kind problem can know as vram faulty ~.~ lucky still can claim warranty.
